Assets/Input_Test/Created by @jia.txt

```diff
@@ -0,0 +1 @@
+Please contact jia@unity3d.com for any concern/comment or poke me on Slack.
```

Comment:
Some quick instructions and what-is-this would be great I think. So far, the QA material is mostly an assortment of what appears like random scenes to the user.